learn-ocaml.el

```diff
@@ -541,6 +526,24 @@ Argument SECRET may be needed by the server."
               (message-box "Token changed succesfully")
               (learn-ocaml-show-metadata))))))))
 
+(defun learn-ocaml-change-server ()
+  "Interactively change the server url."
+  (interactive)
+  (learn-ocaml-give-server-cmd
+   (lambda (s)
+     (if (learn-ocaml-yes-or-no
+          (concat "The current configured server is: " s "\n Do you want to change it?"))
+         (let ((server (read-string "Enter server URL: " "https://")))
+           (learn-ocaml-use-metadata-cmd
+            nil
+            server
+            (lambda (_)
+              (message-box "Server changed succesfully")
+              (learn-ocaml-change-token)
+              (learn-ocaml-show-metadata))))))))
```

Comment:
it seems you move the location of the function `learn-ocaml-change-server` so that it is **after** the `learn-ocaml-change-token` definition, but FYI it is unneeded:
in an Elisp file, the order of the functions is irrelevant (a bit like in JavaScript :)

Reply:
Ok, good to know :)